### PR TITLE
166152330 Export workout

### PR DESCRIPTION
**Pull request by misocho**
#### Title
Adds export/import workouts feature

#### Description

The import/export feature would be used to allow the user to send their workouts to other users so they can add them in their workout schedules.

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue) 

- [x] New feature (non-breaking change which adds functionality) 

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) 

- [ ] This change requires a documentation update

#### How Has This Been Tested?
- [x] Unit tests

#### Checklist:
- [x] Add export workout feature
- [x] Add import workout feature

#### PT stories
[#166152330](https://www.pivotaltracker.com/story/show/166152330)

#### Screenshots
Click the export button to export a JSON file. To import, click the import button and select an exported JSON file
Screenshot 2019-06-13 at 19 13 31

### CODE REVIEW

**Review by Kimaiyo077 — APPROVED** [review]
It works as expected and I was also able to import the workout. Good work @misocho

File: wger/core/static/js/wger-core.js [hunk 1]
```diff
@@ -697,4 +719,35 @@ $(document).ready(function () {
...
+      fetch(url, {
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json',
+          'X-CSRFToken': getCookie('csrftoken')
+        },
+        method: 'POST',
+        body: JSON.stringify(content)
+      }).then(function (res) {
+        console.log(res.json());
```
Comment (martinMutuma) [inline wger/core/static/js/wger-core.js]: Good work. Kindly remove the console.log

**Review by martinMutuma — CHANGES_REQUESTED** [review]
Awesome work. Kindly catch errors for Invalid json file

**Comment by misocho** [comment]
> Awesome work. Kindly catch errors for Invalid json file

@martinMutuma  Changes have been made

**Review by martinMutuma — APPROVED** [review]
Works good now

**Review by bencyn — COMMENTED** [review]
The export feature works as expected. Kindly explain the import process in your PR description as it's not clear how to go about.

**Comment by misocho** [comment]
> The export feature works as expected. Kindly explain the import process in your PR description as it's not clear how to go about.

@bencyn  Done 👍🏾

**Review by bencyn — APPROVED** [review]
Great job works as expected.

**Review by justMuriithi — APPROVED** [review]
Great job. The ease with which I can import or export a workout as JSON is amazing.

**Review by michael-basweti — APPROVED** [review]
Meets all the expectations

**Pull request merged**